src/Presentation/RestApi/EventListener/JsonRequestListener.php

```diff
@@ -47,6 +66,15 @@ public function onKernelRequest(RequestEvent $event): void
 
         try {
             $data = json_decode((string)$request->getContent(), true, 512, \JSON_THROW_ON_ERROR);
+            $error = $this->jsonSchemaValidator->validate($data);
+            if ($error) {
+                $event->setResponse(
```

Comment:
when you set response Does it stop to run action?

Reply:
It stops propagation of the event to further event listeners.